openpype/settings/entities/schemas/projects_schema/schema_project_maya.json

```diff
@@ -10,6 +10,41 @@
             "key": "open_workfile_post_initialization",
             "label": "Open Workfile Post Initialization"
         },
+        {
+            "type": "dict",
+            "key": "explicit_plugins_loading",
+            "label": "Explicit Plugins Loading",
+            "collapsible": true,
+            "is_group": true,
+            "checkbox_key": "enabled",
+            "children": [
+                {
+                    "type": "boolean",
+                    "key": "enabled",
+                    "label": "Enabled"
+                },
+                {
+                    "type": "list",
+                    "key": "plugins_to_load",
+                    "label": "Plugins To Load",
+                    "object_type": {
+                        "type": "dict",
+                        "children": [
+                          {
+                              "type": "boolean",
+                              "key": "enabled",
+                              "label": "Enabled"
+                          },
```

Comment:
I personally think it's very weird this has an enabled state. I'd have expected any entry to be in there to just be intended for auto loading.

Reply:
I experimented with just using a `dict-modifiable` but the plugin names got cut off which is a limitation of the current settings. Not something we should be spending time on implementing due to settings being transitioned to Ayon.
![Capture (1)](https://user-images.githubusercontent.com/1860085/227899985-8b4c93a8-0b82-4b92-a26e-fa8174be9b58.png)

Reply:
I meant more - it could just be a list of strings which are just plug-in names right? It doesn't actually need the toggle. Don't want to autoload then remove the entry? Or is that maybe adding too much burden on the administrator? Hmm, maybe it is.

Reply:
The idea was to provide all the standard plugins, then if the studio has custom plugins, they should be technically minded enough to add the plugin.

@mkolar @antirotor what do you think?

Reply:
The toggles make sense here. It makes it easier to compare multiple projects. You can define all of the available plugins in the default studio settings and when you're overriding it for a project, it it much easier to toggle on and off, rather than delete and then go hunting for what it was that you want to add back. Ending up in a different order than other project increasing the complexity of the search.